commands/logout.go

```diff
@@ -42,33 +40,30 @@ type logoutArguments struct {
 }
 
 func defaultLogoutArguments() logoutArguments {
+	endpoint := config.Config.ChooseEndpoint(cmdAPIEndpoint)
+	token := config.Config.ChooseToken(endpoint, cmdToken)
+
 	return logoutArguments{
-		apiEndpoint: cmdAPIEndpoint,
-		token:       cmdToken,
+		apiEndpoint: endpoint,
+		token:       token,
 	}
 }
 
 func init() {
 	RootCommand.AddCommand(LogoutCommand)
 }
 
-// TODO: separate validation and validation result output
-func logoutValidationOutput(cmd *cobra.Command, args []string) error {
+func logoutValidationOutput(cmd *cobra.Command, args []string) {
 	if config.Config.Token == "" && cmdToken == "" {
-		return errors.New("You are not logged in")
+		fmt.Println("You weren't logged in here, but better be safe than sorry.")
+		os.Exit(1)
```

Comment:
Why is it better to exit immediately than return error?

Reply:
Because this we way can influence the error output shown to the user. If we would return an error here, cobra would show the error text and the entire command usage.